ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml

```diff
@@ -1,193 +0,0 @@
-- dockerfile_literal: |
-    FROM src
-    COPY yq /usr/bin/yq
-    RUN yum install -y --nodocs python3-devel && \
-        pip3 install pygithub GitPython && \
-        dnf clean all && \
-        rm -rf /var/cache/dnf/*
-  from: src
-  inputs:
-    yq-cli:
-      paths:
-      - destination_dir: .
-        source_path: /usr/bin/yq
-  to: microshift-rebase
-releases:
-  arm64-latest:
-    candidate:
-      architecture: arm64
-      product: ocp
-      stream: nightly
-      version: "4.13"
-  latest:
-    candidate:
-      product: ocp
-      stream: nightly
-      version: "4.13"
-resources:
-  '*':
-    limits:
-      memory: 4Gi
-    requests:
-      cpu: 100m
-      memory: 800Mi
-tests:
-- as: copr-rpm-health-check
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-copr-install
-- as: test-unit
-  commands: make test-unit
-  container:
-    from: src
-  cron: 0 0 * * *
-- as: test-srpm
-  commands: make srpm
-  container:
-    from: src
-  cron: 0 0 * * *
-- as: e2e-openshift-conformance-sig-api-machinery
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-api-machinery
-- as: e2e-openshift-conformance-sig-apps
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-apps
-- as: e2e-openshift-conformance-sig-arch
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-arch
-- as: e2e-openshift-conformance-sig-auth
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-auth
-- as: e2e-openshift-conformance-sig-cli
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-cli
-- as: e2e-openshift-conformance-sig-instrumentation
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-instrumentation
-- as: e2e-openshift-conformance-sig-network
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-network
-- as: e2e-openshift-conformance-sig-node
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-node
-- as: e2e-openshift-conformance-sig-scheduling
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-scheduling
-- as: e2e-openshift-conformance-sig-storage
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-storage
-- as: e2e-reboot
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-reboot-tests
-- as: rebase-on-nightlies
-  cron: 0 5 * * *
-  steps:
-    workflow: openshift-microshift-rebase
```

Comment:
I think we might want that

Reply:
Oops you are right. We also need this on 4.12, right?